pages/monsters/index.vue

```diff
@@ -143,4 +187,28 @@ const {
   prevPage,
   nextPage,
 } = paginator;
+
+const encounterStore = useEncounterStore();
+
+const addToEncounter = (monster: any) => {
```

Comment:
Don't love that this needs to be defined on each page but wasn't sure how best to abstract it out.